tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/NLogTests.cs

```diff
@@ -44,6 +45,41 @@ public static IEnumerable<object[]> GetTestData()
             }
         }
 
+        public static IEnumerable<object[]> GetContextTestData()
+        {
+            var minScopeContext = new Version("5.0.0");
+            var minMdlc = new Version("4.6.0");
+            var minMdc = new Version("4.0.0");
+            foreach (var item in PackageVersions.NLog)
+            {
+                if (item.Length < 1) { continue; }
+
+                var version = (string)item[0] == string.Empty ?
+                                  new Version("5.0.0") : // DEFAULT_SAMPLES
```

Comment:
Don't really like this, but I can't think of a better way to make this work

Reply:
Meh, it'll do, there's no good options here 😄